<launch>
    <arg name="ips_to_monitor" default="192.168.1.21"/>

    <node pkg="lost_comms_recovery" type="lost_comms_recovery" name="lost_comms_recovery" clear_params="true" output="screen" respawn="true" >
      <param name="ping_fail_count" value="1"/>
      <param name="ips_to_monitor" value="$(arg ips_to_monitor)"/>
      <remap from="cmd_vel" to="cmd_vel" /> <!-- zero motors -->
      <remap from="arm_mux" to="arm_mux" /> <!-- zero joystick -->
    </node>
</launch>
